pkg/project/operations.go

```diff
@@ -190,37 +191,45 @@ func List(ctx context.Context, opts Options) (projects []string, warnings map[st
 	return projects, warnings, nil
 }
 
-func GetDetails(ctx context.Context, opts Options, projectNames []string) (result []DetailProject, err error) {
-	entryCh := make(chan DetailProject)
-	errCh := make(chan error)
+func GetDetails(ctx context.Context, opts Options, projectNames []string) (projects []DetailProject, err error) {
```

Comment:
Ah, forgot to post this comment:

This function always returns a `nil` error, I think it's safe to drop the `error` return param.